classes/property.py

```diff
@@ -46,8 +46,6 @@ def property_value(property, data):
 
 def random_value(values):
     if values:
-        v = random()
-        for (value, cutoff) in values:
-            if v < cutoff:
-                return value
+        cutoff = random()
+        return next(value for value, limit in values if cutoff >= limit, None)
```

Comment:
This function doesn't so much return a random value as create a random cutoff and select the first value whose limit exceeds that value. So it's `next_value_above_random_threshold` or something like that. The name then suggests that the function is not living its best life, and might prefer to be `next_value_above_threshold` and take a `threshold`, which could be random, or fixed, or whatever you like. 

Since we don't expect this to stick around, I'm not too concerned about it.